src/java/org/apache/cassandra/auth/MutualTlsWithPasswordFallbackAuthenticator.java

```diff
@@ -34,6 +37,9 @@
 public class MutualTlsWithPasswordFallbackAuthenticator extends PasswordAuthenticator
 {
     private final MutualTlsAuthenticator mutualTlsAuthenticator;
+
+    private static final Set<String> MODES = Sets.newHashSet(PasswordAuthenticator.MODE, MutualTlsAuthenticator.MODE);
```

Comment:
I am just wondering ... why are we referencing these String constants in either authenticators? I think that the more "clear" approach would be to return in `getSupportedAuthenticationModes` `PasswordAuthenticator.MODES` and `MutualTlsAuthenticator.MODES`. In other words, we should create a new set by combination of `MODES`. That way, when some `MODE` would be added to these authenticators, this one would already return them all without any need to do add it here.

Reply:
agreed, changed this :+1: